composer.json

```diff
@@ -6,11 +6,11 @@
         "php-di/php-di": "^5.4",
         "aws/aws-sdk-php": "^3.19",
         "symfony/yaml": ">=2.3",
-        "symfony/console": ">=2.6",
-        "symfony/event-dispatcher": ">=2.5",
         "ericpoe/haystack": "^1.0",
         "guzzlehttp/guzzle": "^6.2",
-        "psr/log": "^1.0"
+        "psr/log": "^1.0",
+        "symfony/console": "^4.4",
+        "symfony/event-dispatcher": "^4.0"
     },
     "require-dev": {
```

Comment:
We should probably add sort-packages config to all our projects to help with diffs

https://stackoverflow.com/a/52852369